### PR TITLE
fancy awesome-npx render instructions

### PR DESCRIPTION
**Pull request by zkat**
using this command lets you check out the readme in a pager _and_ render it pretty!

- [x] I have read & abide by the CONTRIBUTING.md and CODE_OF_CONDUCT.md guidelines and agree to license my contributions as CC0-1.0.

type of change
=====
- [ ] adding a tool / package / resource
  - please keep items sorted alphabetically
- [ ] a minor correction (spelling, broken link, name change)
- [ ] re-organization (please discuss on an issue before hand!)
- [x] other

additional notes
=====

shiny wow

### CODE REVIEW

**Comment by zkat** [comment]
Note: in retrospect, please don't merge this yet. I just fixed a but in npx where `npx foo | npx bar` was failing -- if you wait until the next npm release with npx in it (frankly, probably tomorrow/wednesday), then this command will _actually_ work. And it'll still work if anyone updates to the npx I'm about to release.

**Comment by junosuarez** [comment]
ah, gotcha- I was just testing this out and experiencing that not working :) `lessmd` is really cool though (colorfullness++) and I'd accept a PR adding it!

**Comment by junosuarez** [comment]
I can confirm that colored output is working correctly in npx@9.1.0 for `npx awesome-npx | npx lessmd`

I don't like having the extra `| less` at the end, and in any case the `less` I have on my machine (macos) doesn't behave well with the color escape codes, so I opened a PR on the lessmd package: https://github.com/linuxenko/lessmd/issues/5

(this is what I get otherwise, which isn't a very good first experience for someone trying out `npx` imho):
image

**Comment by zkat** [comment]
Also tbh I'm _pretty_ sure the reason that last `| less` is needed is because of a `lessmd` bug? I can't get it to paginate for me, even though the project claims it does?

**Comment by junosuarez** [comment]
it only goes into pager mode from a file, not from stdin- that's what https://github.com/linuxenko/lessmd/issues/5 is about

**Comment by junosuarez** [comment]
(it's worth noting that its pager is much less fully-featured than `less`, fwiw - now accepting PRs for a fully npx-able version of `less` that also supports emoji and tty colors! 😄 )

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ npx: installed 1 in 1.413s
 
 you can even use `npx` to view this list of `awesome-npx` tools:
 ```
-> npx awesome-npx
+> npx awesome-npx | npx lessmd
```
Comment (alvaropinot) [inline README.md:26]: 🤔 Would this be better inside the `cli.js` it self and the `package.json` as a dep?  So we could still run only `npx awesome-npx`, `npx` will take care of the deps 😛

Comment (junosuarez) [reply README.md:26]: actually, that's a really good idea. If you're willing, I'd love to take a PR for that, otherwise please make an issue :)

Comment (zkat) [reply README.md:26]: I like this idea too.

I'm gonna close this PR so you can go ahead with that 👍

**Pull request closed without merging**